rest/src/main/java/at/jku/isse/ecco/rest/Settings.java

```diff
@@ -27,4 +34,12 @@ private static String getAutoLocation() {
             }
         }
     }
+
+    private static Boolean isRunningInsideDocker() {
+        try (Stream<String> stream = Files.lines(Paths.get("/proc/1/cgroup"))) {
```

Comment:
this check will fail if the host is using cgroup v2.
first check if the file "/.dockerenv" exists (see https://github.com/localstack/localstack/pull/4352/files)
if that check fails the check you already have, i.e., looking for "/docker" inside the cgroup v1 path.

I would also suggest looking for a library, there might be already someone that has solved this problem